addons/event_booth_sale/models/sale_order_line.py

```diff
@@ -95,11 +95,6 @@ def _update_event_booths(self, set_paid=False):
                 so_line.event_booth_ids.sudo().action_set_paid()
         return True
 
-    def _get_sale_order_line_multiline_description_sale(self):
-        if self.event_booth_pending_ids:
-            return self.event_booth_pending_ids._get_booth_multiline_description()
```

Comment:
Maybe check if `self.event_booth_pending_ids != self._origin.event_booth_pending_ids` this way we only recompute it when there are effectively different booths